### PR TITLE
Improve documentation

### PR DESCRIPTION
**Pull request by rubenrails**
# DON NOT MERGE...YET
Related #86 

This PR aims for improving documentation for the current implementation. DO NOT add/change any functionallity, only documentation changes will be merged.

### CODE REVIEW

**Comment by rubenrails** [comment]
@seancookr @javierg @netlogics @norcal82 This PR's intend is to kick off the documentation improvements, feel free to add commits to this PR.

File: README.md [hunk 1]
```diff
@@ -71,8 +71,9 @@ User.all #Dolly::Collection #<#User...>, <#User...>
 user = User.find "a1b2d3e" #<#User...>
 user.name
 
-#Return a User object based on the custom view.
-user = User.view 'view_name', {key: ["a", "b", "c"], reduce: true}
+#Return a User object based on the custom view. `include_docs` is always true for this method.
```
Comment (netlogics) [inline README.md]: Is a `User` object returned or a json string?

Comment (rubenrails) [inline README.md]: JSON String, that's why I added the `from_json` call, as it previously seemed to indicate you'd be getting a User instead of a String.

Comment (netlogics) [inline README.md]: Gotcha.

**Comment by seancookr** [comment]
@rubenrails Since no real has been made for this, I think we should merge this so as to not let it get stale

**Comment by javierg** [comment]
:shipit:

**Pull request merged**